package.json

```diff
@@ -75,17 +76,17 @@
     "husky-run": "^0.0.0",
     "lint-staged": "^12.3.7",
     "prettier": "^2.6.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "rimraf": "^3.0.2",
     "rollup": "^2.70.1",
     "rollup-plugin-filesize": "^9.1.2",
-    "rollup-plugin-terser": "^7.0.2",
-    "typescript": "^4.6.3"
+    "three": "^0.150.1",
+    "typescript": "^4.9.5"
   },
   "peerDependencies": {
-    "@react-three/fiber": ">=7.0",
-    "react": ">=17.0",
-    "three": ">=0.136.0"
+    "@react-three/fiber": ">=8.0",
+    "react": ">=18.0",
+    "three": ">=0.150.1"
```

Comment:
Note that this is major semver, so this is an excuse for intrusive changes. I'd also try to keep three.js as low as your dependencies allow -- postprocessing ranges from `0.138` and newer.

Reply:
latest postprocessing has breaking changes, with different names and deprecated effects

Reply:
Changed to `>= 0.138.0 < 0.152.0` like in pp